src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
+  const handleDelete = event => {
+    const activityId = event.target.dataset.id;
+    onHandleDelete(activityId);
+  };
+  return (
+    <>
+      <div>
+        <label>Aufgaben des Tages</label>
+      </div>
+      {activities.map(activity => (
+        <div key={activity.id}>
+          <StyledActivitiList>
+            <label>Dein Tag:</label>
```

Comment:
The `label` element is used for interactive elements such as inputs; I think you want to use a heading here.